users/management/commands/load_initial_data.py

```diff
@@ -127,6 +128,15 @@ def handle(self, *args, **options):
         self._add_member_to_project_if_not_added_yet(project_terminated, user_manager_1)
         self._add_member_to_project_if_not_added_yet(project_terminated, user_manager_2)
 
+        #
+        # Task activities
+        #
+
+        task_activity_review = TaskActivityType.objects.get(name="Review")
```

Comment:
If for some reason those objects would not exist (ie. someone will run `python manage.py flush` to clear database) this code will fail, so I think it is better to use `objects.get_or_create()` here and below.